common4j/src/main/com/microsoft/identity/common/java/crypto/KeyAccessorStringAdapter.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.crypto;
+
+import com.microsoft.identity.common.java.exception.ClientException;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NonNull;
+
+import static com.microsoft.identity.common.java.AuthenticationConstants.ENCODING_UTF8;
+
+/**
+ * Add helper functions which takes in parameter or produce results in a ready-to-store (String) form.
+ */
+@AllArgsConstructor
+public class KeyAccessorStringAdapter {
+
+    public IKeyAccessor mKeyAcccesor;
+
+    public String encrypt(@NonNull String plainText) throws ClientException{
```

Comment:
no javadoc, makes no statement on the encryption type/cipher/alg -- since this is delegating to a `public` member its not clear what's happening, who sets it, and when